core/src/main/resources/hudson/model/ComputerSet/index.jelly

```diff
@@ -30,21 +30,22 @@ THE SOFTWARE.
 <l:layout title="${%Nodes}">
   <st:include page="sidepanel.jelly" />
   <l:main-panel>
-    <j:set var="monitors" value="${it._monitors}"/>
-    <j:set var="tableWidth" value="${3}"/>
-
-    <div class="jenkins-app-bar">
+    <div class="jenkins-app-bar jenkins-app-bar--no-border">
       <div class="jenkins-app-bar__content">
-        <h1>${%Manage nodes and clouds}</h1>
+        <h1>${%Nodes}</h1>
       </div>
-      <l:hasAdministerOrManage>
-        <div class="jenkins-app-bar__controls">
+      <div class="jenkins-app-bar__controls">
```

Comment:
Similarly, this seems actively hostile to plugin developers to not have a Jelly control.

Also, I still don't understand the naming convention. Do we document anywhere when to use `-`, `--`, `__`, or `-!-` as separator?